### DISCUSSION TITLE
First post - Konstantinos Chousos

### DISCUSSION THREAD

**Discussion by giscus** [kchousos/kchousos.github.io#1] [Announcements]
# First post - Konstantinos Chousos

This is the first post of the site. Really just a test to see if everything is working correctly.
Placeholder text
The following is random placeholder text.
The first radio series of six episodes (called “Fits” after the names of the sections of Lewis Carroll’s nonsense poem “The Hunting of the Snark”)[15] was broadcast in 1978 on BBC Radio 4. Despite a low-key launch of the series (the first episode was broadcast at 10:30 pm on Wednesday, 8 March 1978), it received generally good reviews and a tremendous audience reaction for radio.

http://kchousos.github.io/posts/test-post/

**Comment by kchousos** [comment]
Testing comments and markdown rendering.

# H1
 
Lorem ipsum dolor sit amet, *consectetur* adipisicing elit, sed do eiusmod
tempor incididunt ut **labore et dolore magna aliqua**. Ut enim ad minim veniam,
 
 
> quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
 
consequat. ***Duis aute irure dolor*** in reprehenderit in voluptate velit esse
cillum dolore eu fugiat nulla pariatur. ~~Excepteur sint occaecat~~ cupidatat nonproident, sunt in culpa qui officia deserunt mollit anim id est laborum.
 
## H2
 
Lorem ipsum dolor sit amet, *consectetur* adipisicing elit, sed do eiusmod
tempor incididunt ut **labore et dolore magna aliqua**. Ut enim ad minim veniam,
quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
consequat. 
 
---
 
***Duis aute irure dolor*** in reprehenderit in voluptate velit esse
cillum dolore eu fugiat nulla pariatur. ~~Excepteur sint occaecat~~ cupidatat non
proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
 
### H3
 
unordered list:
 
* item-1
  * sub-item-1
  * sub-item-2
- item-2
  - sub-item-3
  - sub-item-4
+ item-3
  + sub-item-5
  + sub-item-6
 
 
ordered list:
 
1. item-1
   1. sub-item-1
   2. sub-item-2
2. item-2
   1. sub-item-3
   2. sub-item-4
3. item-3
 
#### Header4
 
Table Header-1 | Table Header-2 | Table Header-3
:--- | :---: | ---:
Table Data-1 | Table Data-2 | Table Data-3
TD-4 | Td-5 | TD-6
Table Data-7 | Table Data-8 | Table Data-9
 
##### Header5
 
You may also want some images right in here like ![GitHub Logo](https://cloud.githubusercontent.com/assets/5456665/13322882/e74f6626-dc00-11e5-921d-f6d024a01eaa.png "GitHub") - you can do that but I would recommend you to use the component "image" and simply split your text.
 
###### Header6
 
Let us do some links - this for example: https://github.com/MinhasKamal/github-markdown-syntax is **NOT** a link but this: is [GitHub](https://github.com/MinhasKamal/github-markdown-syntax)
